### PR TITLE
Enable Present Card Payments in release build

### PR DESCRIPTION
**Pull request by malinajirka**
Parent issue #3724

Note: All the reviewers were added automatically by GitHub. I'm not sure why it requested a review from some of you explicitly, I expected it to request one review from `android developers` and one review from `platform 9`. 

This PR enables Card Present Payments in release build
- moves classes from `debug` package to `main` package
- moves "location permission" from debug AndroidManifest to CardReaderModule's main manifest
- enables the feature flag in non-debug builds
- updates CI config so CardReaderModule's unit tests are run on the release version

To test:
Connect to a reader in both debug and release builds.

Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Review by kidinov — APPROVED** [review]
The code looks and works well in both debug and release builds. I don't merge it because probably nice to have an approval from someone from the p9 team

File: .circleci/config.yml [hunk 1]
```diff
@@ -71,7 +71,7 @@ jobs:
           command: ./gradlew --stacktrace testVanillaRelease
       - run:
           name: Unit tests CardReaderModule
-          command: ./gradlew --stacktrace lib:cardreader:testDebug
+          command: ./gradlew --stacktrace lib:cardreader:testRelease
```
Comment (jkmassel) [inline .circleci/config.yml:74]: For tests, should we be keeping this as `testDebug`? Or should we be testing both types of build?

Comment (malinajirka) [reply .circleci/config.yml:74]: The tests in the main module use `release` as well, so I wanted to stay consistent. The reason why we started with `testDebug` was that the feature was not available in release builds. Wdyt?

Comment (malinajirka) [reply .circleci/config.yml:74]: If we want to run all tests on debug builds, I'd suggest making this change in a separate PR across all the repos. Wdyt?

Comment (malinajirka) [reply .circleci/config.yml:74]: In order to avoid conflicts I'm merging this PR -> we can update the code later if needed. Let me know @jkmassel.

**Pull request merged**